core/domain/exp_jobs_one_off.py

```diff
@@ -325,7 +325,10 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         if key == 'SUCCESS':
-            yield (key, values[:VALID_MATH_INPUTS_YIELD_LIMIT])
+            if values[0] == 'None':
+                yield (key, len(values))
+            else:
+                yield (key, values[:VALID_MATH_INPUTS_YIELD_LIMIT])
```

Comment:
Why do you still have a limit? Don't do that.

Reply:
I've removed it now. I thought we just need to see a sample of the output to verify manually.